apps/remix-ide/src/app/tabs/runTab/contractDropdown.js

```diff
@@ -385,7 +388,10 @@ class ContractDropdownUI {
   loadFromAddress () {
     this.event.trigger('clearInstance')
 
-    var address = this.atAddressButtonInput.value
+    let address = this.atAddressButtonInput.value
+    if (!ethJSUtil.isValidChecksumAddress(address)) {
+      address = ethJSUtil.toChecksumAddress(address)
```

Comment:
Are we not planning to show checksummed address in the input?

Reply:
no